research/src/pages/components/selectmenu.mdx

```diff
@@ -0,0 +1,257 @@
+---
+name: selectmenu explainer
+path: /components/selectmenu
+showInMenu: true
```

Comment:
@josepharhar , you'll need to import the layout here

Reply:
it'll look something like this:
`layout: ../../layouts/ComponentLayout.astro`

Reply:
done